cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/BufferingTable.java

```diff
@@ -496,9 +496,8 @@ private void putInternal(byte[] row, byte[][] columns, byte[][] values) {
   }
 
   /**
-   * NOTE: Depending on the use-case, calling this method may be much less
-   *       efficient than calling same method with columns as parameters because it may always require round trip to
-   *       persistent store
+   * NOTE: Depending on the use-case, calling this method may be much less efficient than calling same method
+   *       with columns as parameters because it may always require round trip to persistent store.
```

Comment:
This is a pre-existing comment, but I believe the reason is that the existing set of rows has to be read, and then a delete marker needs to be `PUT` to those columns (can't simply use hbase delete on the entire row).